File: moban_java/src/main/java/org/jdk/project/dto/config/RobotUpsertRequest.java
package org.jdk.project.dto.config;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class RobotUpsertRequest {
  private Long loungeId;
  private Long regionId;
  private String robotId;
  private String robotName;
  private String mac;
  private String robotIp;
  private String robotType;
  private Integer batteryState;
  private String chargingState;
  private String workingState;
  private String standbyState;
  private String positioningState;
  private Integer enable;
  private String oriCoordinate;
  private Boolean adminMode;
  private String errorCode;
  private String errorMessage;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/RobotUpsertRequest.kt
package org.jdk.project.dto.config

data class RobotUpsertRequest(
    var siteId: Long? = null,
    var pointId: Long? = null,
    var robotId: String? = null,
    var robotName: String? = null,
    var mac: String? = null,
    var robotIp: String? = null,
    var robotType: String? = null,
    var batteryState: Int? = null,
    var chargingState: String? = null,
    var workingState: String? = null,
    var standbyState: String? = null,
    var positioningState: String? = null,
    var enable: Int? = null,
    var oriCoordinate: String? = null,
    var adminMode: Boolean? = null,
    var errorCode: String? = null,
    var errorMessage: String? = null,
    var remark: String? = null,
)
